src/components/new-safe/create/steps/ReviewStep/index.tsx

```diff
@@ -203,46 +218,52 @@ const ReviewStep = ({ data, onSubmit, onBack, setStep }: StepRenderProps<NewSafe
         </Grid>
       </Box>
 
-      <Divider />
-      <Box className={layoutCss.row} display="flex" flexDirection="column" gap={3}>
-        {canRelay && !isSocialLogin && (
-          <Grid container spacing={3}>
-            <ReviewRow
-              name="Execution method"
-              value={
-                <ExecutionMethodSelector
-                  executionMethod={executionMethod}
-                  setExecutionMethod={setExecutionMethod}
-                  relays={minRelays}
+      {!isCounterfactual && (
+        <>
+          <Divider />
+          <Box className={layoutCss.row} display="flex" flexDirection="column" gap={3}>
+            {canRelay && !isSocialLogin && (
+              <Grid container spacing={3}>
+                <ReviewRow
+                  name="Execution method"
+                  value={
+                    <ExecutionMethodSelector
+                      executionMethod={executionMethod}
+                      setExecutionMethod={setExecutionMethod}
+                      relays={minRelays}
+                    />
+                  }
                 />
-              }
-            />
-          </Grid>
-        )}
+              </Grid>
+            )}
 
-        <Grid data-testid="network-fee-section" container spacing={3}>
-          <ReviewRow
-            name="Est. network fee"
-            value={
-              <>
-                <NetworkFee totalFee={totalFee} willRelay={willRelay} chain={chain} />
-
-                {!willRelay && !isSocialLogin && (
-                  <Typography variant="body2" color="text.secondary" mt={1}>
-                    You will have to confirm a transaction with your connected wallet.
-                  </Typography>
-                )}
-              </>
-            }
-          />
-        </Grid>
+            <Grid data-testid="network-fee-section" container spacing={3}>
+              <ReviewRow
+                name="Est. network fee"
```

Comment:
Don't we want to also display the network fee if the users deploys later?
Otherwise users might transfer funds into their Safe and only then learn how much the deployment actually costs.

Reply:
As part of https://github.com/safe-global/safe-wallet-web/issues/3210 we will add a Pay now / Pay later option with some educational text.